docs/ai.md

```diff
@@ -46,3 +46,47 @@ can use the vector database to find relevant documents (technically, only some
 parts of the documents called chunks). Those documents are sent back to
 LibreChat that can be added to the prompt, so that the LLM can use them as a
 context when answering.
+
+### GET /ai/open
+
+This route returns the parameters to open an iframe with OpenWebUI. It creates
+an account on the OpenWebUI chat server if needed, creates a token an returns
+the configured URL of the OpenWebUI chat server.
+
+### Request
+
+```http
+GET /ai/open HTTP/1.1
+```
+
+### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+```
+
+```json
+{
+  "data": {
+    "type": "io.cozy.ai.url",
```

Comment:
Isn't it a bit too generic to use `url` as the last part here since this is only the frontend part and we might have other AI URLs ?
We kinda already do with the external indexers even if they don't have their own doctype for now.

Something like `io.cozy.ai.chatbot` or `io.cozy.ai.open` would be more appropriate in my opinion.